config/core/test-configs.yaml

```diff
@@ -2016,6 +2018,12 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-livepatch
+      - kselftest-lkdtm
+      - kselftest-seccomp
```

Comment:
I'm not sure there are enough imx6q-sabrelite available to cover all this.  See this earlier commit: 3578f05e6cb9be3a8de9b25bb7e8578a227111a7

```
commit 3578f05e6cb9be3a8de9b25bb7e8578a227111a7
Author: Guillaume Tucker <guillaume.tucker@collabora.com>
Date:   Wed May 26 16:35:21 2021 +0100

    test-configs.yaml: only run baseline on imx6q-sabrelite
    
    There aren't enough imx6q-sabrelite boards available in the Collabora
    lab to run all the test plans that were assigned to it, so reduce to
    only baseline until this is resolved with more hardware.  Enable
    baseline-nfs to keep running only in lab-baylibre in the meantime.
    
    Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>
```

So it's probably better to only keep running baseline on it.

Reply:
I've found 9 imx6q-sabrelite in Collabora's Lava: https://lava.collabora.co.uk/scheduler/alldevices/active?search=&length=100&page=1#table Maybe there have been added recently?

Reply:
Ah yes, good point. Let's check the results and the queue size with the staging jobs to get an idea.